README.md

```diff
@@ -45,15 +59,36 @@ To test the second store (alone or in conjunction) change the metric path to `te
 Record structure
 ----------------
 
-The only supported protocol is [line](https://graphite.readthedocs.io/en/latest/feeding-carbon.html#the-plaintext-protocol). The records should follow the structure:
+The only supported protocol is [line](https://graphite.readthedocs.io/en/latest/feeding-carbon.html#the-plaintext-protocol). The records have the structure:
 ```
 path.path.path value datetime
 ```
 
 Config
 ------
 
-Please refer to the sample configs in the _config_ folder for examples and documentation.
+Please refer to the [sample config](config/config.toml) for examples and documentation.
+
+Routing
+-------
+
+Is defined in the [rules config](config/rules.toml) that is in turn referred to in the [main config](config/config.toml). This is how it works:
+- routing rules are applied to each incoming record in order;
+- `continue` variable defines whether to continue matching next rules. It is `true` by default;
```

Comment:
is it really defaults to true?
In our production config it used differently with assumption that it defaults to 'false'

Reply:
That's right. Updated.

For findings like this one is why we write the docs.